### PR TITLE
Fix #5125 - Removed prerelease field from Addon model and added migration 946

### LINKED ISSUE #5125: remove Addon.prerelease column?
I can't tell much about how it's supposed to be used but it's apparently unused so let's get rid of it if possible.

### PR DESCRIPTION
**Pull request by harikishen**
I have removed the `prerelease` field from the `Addon` model. I have also written the corresponding [migration](https://github.com/harikishen/addons-server/blob/ticket_5125/src/olympia/migrations/946-remove-prerelease-field-addons.sql). The fix also required the updation of several other json files.

### CODE REVIEW

**Comment by harikishen** [comment]
@EnTeQuAk

File: tmp/addons/.gitignore [hunk 1]
```diff
@@ -1 +0,0 @@
-*
```
Comment (eviljeff) [inline tmp/addons/.gitignore]: Why have you deleted all of these folders?  Mistake?

Comment (harikishen) [inline tmp/addons/.gitignore]: i think that was a mistake

Comment (harikishen) [inline tmp/addons/.gitignore]: there was some problem with the jetpack file on my mac earlier on ...i deleted some files then

Comment (harikishen) [inline tmp/addons/.gitignore]: @eviljeff  i'm guessing those files don't affect the merge right ? They're marked to be ignored in `.gitignore`

Comment (eviljeff) [inline tmp/addons/.gitignore]: Can you revert the deletions.  It would affect the merge.

Comment (harikishen) [inline tmp/addons/.gitignore]: i'll do that

**Comment by harikishen** [comment]
i've fixed those

**Pull request merged**